tests/convert_if_to_select/non_secret_conditionals.mlir

```diff
@@ -0,0 +1,53 @@
+// RUN: heir-opt --convert-if-to-select %s | FileCheck %s
+
+// CHECK-LABEL: @non_secret_condition_outside_of_secret_generic_with_secret_tensor
+func.func @non_secret_condition_outside_of_secret_generic_with_secret_tensor(%inp: !secret.secret<tensor<16xi16>>, %cond: i1) -> !secret.secret<tensor<16xi16>> {
+  // CHECK-NEXT: %[[RESULT:.*]] = secret.generic ins(%[[INP:.*]] : !secret.secret<tensor<[[T:.*]]>>) {
```

Comment:
I won't ask you to undo this, but for future reference it may be stabler, less work, and more readable to reduce these two tests to have a single assertion: `// CHECK-NOT: arith.select`